fcrepo-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -34,6 +34,16 @@
 
 	</servlet-mapping>
 
+  <filter>
+    <filter-name>ETagFilter</filter-name>
+    <filter-class>org.springframework.web.filter.ShallowEtagHeaderFilter</filter-class>
```

Comment:
@awoods because doing so is good practice. It is for much the same reason I refactored the JS in #1070 

And by "small" I meant that it would be presumably faster for the server to compute the checksum by reading the file locally than it would be to send all the bytes over the network.